### ISSUE TITLE
Mobile A11y Testing 11/1

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/iron-collapse#30]
On Android with Talkback, right now, I don't have any sense for the state -- collapsed or expanded. When I double tap with Talkback, I just hear ""collapse #1"" each time, whether expanded or not. This context is missing for the user.

**Comment by valdrinkoshi** [comment]
I did some testing on different devices, I just load the https://elements.polymer-project.org/elements/iron-collapse?view=demo:demo/index.html&active=iron-collapse and put the focus on the first button, and:
- on Mac I hear "collapse n.1, collapsed, button"
- on Android I hear "collapse n.1"
- on iOs I hear "collapse n.1, button, expanded"

**Comment by valdrinkoshi** [comment]
Here a simplified jsbin http://jsbin.com/quduco/9/edit?html,output
On Android Talkback the focus won't be set on opened, and the attribute `aria-expanded` is ignored.
@lpalmaro FYI, do you see any incorrect setup in the jsbin?

**Comment by valdrinkoshi** [comment]
cc/ @alice

**Comment by alice** [comment]
The two mobile cases look like platform limitations to me.

Currently, TalkBack doesn't support anything like `aria-expanded`, so there's nothing to map that property onto. I believe work is in progress to address this.

I would file a bug for the iOS issue via the steps described in https://developer.apple.com/bug-reporting/using-bug-reporter/.

**Comment by valdrinkoshi** [comment]
Closing this as this is a limitation of TalkBack. Will verify again the iOs behavior and file the bug if needed.

**Comment by valdrinkoshi** [comment]
Tested again on ipad with ios 9.2 and I don't face the bug :tada:
